packages/x-charts/src/internals/components/ChartsAxesGradients/ChartsAxesGradients.tsx

```diff
@@ -8,7 +8,7 @@ import ChartsContinuousGradient from './ChartsContinuousGradient';
 export function useChartGradient() {
   const { chartId } = React.useContext(DrawingContext);
   return React.useCallback(
-    (axisId: string, direction: 'x' | 'y') => `${chartId}-graient-${direction}-${axisId}`,
+    (axisId: string, direction: 'x' | 'y') => `${chartId}-gradient-${direction}-${axisId}`,
```

Comment:
This is technically a BC, albeit, so small that, IMHO, we could go ahead with it...
Did you think about it? What do you think? 🤔

Reply:
Hum, I didn't look much into because no docs changed due to this.

Upon further understanding what it does, it doesn't look like it is documented for external use nor it affects any of the tests.

Do you have more concrete examples of what issues this change could create?

Reply:
It's only the fact that an `id` DOM attribute changes and someone could be relying on it, hence, essentially it is a breaking change.
However, as you have also pointed out, it is not documented and essentially nothing changes functionality wise, that's why I mentioned that, IMHO, it is fine to go ahead with this change.
I just asked if you have considered it. 👍 
If it were any more important, we could add a breaking change note in the release changelog, but I'm not sure if it is even worth that. 🙈 😆

Reply:
Do you think we should put a small note in the changelog about it? "Fixed a typo in an undocumented x-charts gradient css class that could have unintended sideeffects"?

Reply:
It won't hurt. :)
However, am I missing something or does it end up in the `id` field instead of a `class`? 🤔 
![Screenshot 2024-04-29 at 16 24 02](https://github.com/mui/mui-x/assets/4941090/a1a50edc-7b46-4ca6-b50f-1a4fdbce7574)

Reply:
You are correct 👍